cmd/snap/cmd_run_test.go

```diff
@@ -482,7 +482,7 @@ func (s *RunSuite) TestSnapRunCreateDataDirs(c *check.C) {
 	c.Assert(err, check.IsNil)
 	info.SideInfo.Revision = snap.R(42)
 
-	err = snaprun.CreateUserDataDirs(info)
+	err = snaprun.CreateUserDataDirs(info, nil)
```

Comment:
we don't seem to have here and many other places tests for when the options aren't nil and the flag is set